content/en/tracing/app_analytics/_index.md

```diff
@@ -306,15 +306,16 @@ tracer.use('mysql', {
 
 Database tracing is not captured by App Analytics by default and you must enable collection manually for each integration. For example, to enable App Analytics for ADO.NET:
 
-* Environment Variable or AppSetting: `DD_ADONET_ANALYTICS_ENABLED=true`
+* Windows: Environment Variable or AppSetting: `DD_ADONET_ANALYTICS_ENABLED=true`
+* Linux and Windows: Environment Variable or AppSetting: `DD_AdoNet_ANALYTICS_ENABLED=true`
```

Comment:
Sorry, I meant go back to a single line:

```suggestion
* Environment Variable or AppSetting: `DD_AdoNet_ANALYTICS_ENABLED=true`
```

Reply:
Thanks for clarifying. That makes sense!